src/types.ts

```diff
@@ -546,3 +558,16 @@ export interface IKVWalletIndex extends IKVStoreIndex<void> {
   cleanAccessData(): Promise<void>;
   cleanWalletData(clear: boolean): Promise<void>;
 }
+
+export interface IKVNanoContractIndex extends IKVStoreIndex<void> {
+  isNanoContractRegistered(ncKey: string): Promise<boolean>;
+  getNanoContract(ncKey: string): Promise<INcData | null>;
+  registerNanoContract(ncKey: string, ncValue: INcData): Promise<void>;
```

Comment:
Is it missing `unregisterNanoContract`?

Reply:
Fixed at https://github.com/HathorNetwork/hathor-wallet-lib/pull/614/commits/f7f228e35c65a12e4ab9eade924898eb8c63348b